Content.Shared/DoAfter/DoAfter.cs

```diff
@@ -18,11 +18,13 @@ public sealed class DoAfter
     public TaskCompletionSource<DoAfterStatus> Tcs;
 
     //TODO: Should be merged into here
+    [DataField("pssEventArgs")]
     public readonly DoAfterEventArgs EventArgs;
```

Comment:
The doafter changes will clash with #14609 which already adds data-fields for everything, so its better to just remove these changes.